source/frontend/src/features/leases/detail/LeasePages/stakeholders/models.ts

```diff
@@ -148,20 +156,21 @@ export class FormStakeholder {
       this.stakeholderType = fromTypeCode(apiModel.stakeholderTypeCode) ?? undefined;
       this.primaryContactId = apiModel.primaryContactId?.toString() ?? undefined;
       this.initialPrimaryContact = apiModel.primaryContact ?? undefined;
-    } else if (exists(selectedContactModel)) {
+    } else if (exists(contactModel)) {
       // In this case, construct a tenant using a contact.
+      const selectedContactModel = contactModel.contact;
 
-      this.id = selectedContactModel?.id;
+      this.id = contactModel.contact?.id;
 
       this.summary = selectedContactModel.summary;
       this.email = selectedContactModel.email;
       this.mailingAddress = { streetAddress1: selectedContactModel.mailingAddress } as FormAddress;
       this.municipalityName = selectedContactModel?.municipalityName;
-      this.note = selectedContactModel.note;
+      this.note = undefined;
```

Comment:
Hmm, can you help me understand this change? Wouldn't these values already be undefined? Why were we populating these fields previously but not any more?